A/armadillo/build_tarballs.jl

```diff
@@ -31,10 +31,14 @@ if [[ "${nbits}" == 64 ]] && [[ "${target}" != aarch64* ]]; then
     done
     if [[ "${target}" == *-apple-* ]] || [[ "${target}" == *-mingw* ]]; then
         FLAGS+=(-DALLOW_OPENBLAS_MACOS=ON)
+    fi
+    if [[ "${target}" == *-linux-* || "${target}" == *-freebsd* ||
```

Comment:
Why this `if`?  Aren't these all the platforms we support anyway?

Reply:
Yeah, I guess it is now; it wasn't when I started. :) I simplified it in 194a9da.  Thanks for pointing that out! :+1: